README.md

```diff
@@ -1,6 +1,8 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
+
+>First Module project.
 
 > One paragraph statement about the project.
```

Comment:
- [ ] You have done an excellent job with your project but I would suggest filling or removing the information for the sections that you're not using/don't apply. Remember that the README is the first document recruiters or team members will see when checking your project and it should be professional and updated.
![image](https://user-images.githubusercontent.com/81522435/143041465-6a5d8472-7757-4881-b2f0-464cd4933ed3.png)
Please, remove the sections that your project is not using to make your README look even better :wink: